docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -27,31 +25,23 @@ Older, unmaintained versions of Ansible can contain unfixed security
 vulnerabilities (*CVE*).
 
 You can refer to the :ref:`porting guides<porting_guides>` for tips on updating your Ansible
-playbooks to run on newer versions.
+playbooks to run on newer versions. You can download the Ansible release from `<https://releases.ansible.com/ansible/>`_.
 
-.. _release_schedule:
 
-Release status
-``````````````
-This table links to the release notes for each major release. These release notes (changelogs) contain the dates and significant changes in each minor release.
-
-==============================      =================================================
-Ansible Release                     Status
-==============================      =================================================
-devel                               In development (2.10 unreleased, trunk)
-`2.9 Release Notes`_                Maintained (security **and** general bug fixes)
-`2.8 Release Notes`_                Maintained (security **and** critical bug fixes)
-`2.7 Release Notes`_                Maintained (security fixes)
-`2.6 Release Notes`_                Unmaintained (end of life)
-`2.5 Release Notes`_                Unmaintained (end of life)
-<2.5                                Unmaintained (end of life)
-==============================      =================================================
-
-You can download the releases from `<https://releases.ansible.com/ansible/>`_.
-
-.. note:: Ansible maintenance continues for 3 releases.  Thus the latest Ansible release receives
-    security and general bug fixes when it is first released, security and critical bug fixes when
-    the next Ansible version is released, and **only** security fixes once the follow on to that version is released.
+This table links to the release notes for each major Ansible release. These release notes (changelogs) contain the dates and significant changes in each minor release.
+
+==================================      =================================================
+Ansible Release                         Status
+==================================      =================================================
+devel                                   In development (2.11 unreleased, trunk)
```

Comment:
How about adding 2.10 here already (as "in development")?

Reply:
added. thanks!